app/src/main/java/at/aau/edu/wizards/ui/lobby/LobbyViewModel.kt

```diff
@@ -29,18 +34,26 @@ class LobbyViewModel(
     fun stopAdvertising() {
         server.stopBroadcasting()
     }
-
+    var numplayer = 1
+    var check_too_many_player = false
```

Comment:
So many warnings here. @Kirschzelle always points them out. Main issue is that they are warnings in sonar. And btw they have a reason to be pointed out by the IDE

Reply:
<img width="459" alt="Screenshot 2023-05-11 at 21 03 47" src="https://github.com/lechld/Wizard/assets/68105512/52f52285-6a06-45d8-b0b5-bf067afc9b08">

Welle drunter --- nicht zum Spaß da

Reply:
please resolve conversations when you fixed them